scripts/prompts.ts

```diff
@@ -195,13 +243,29 @@ export async function promptRelease(opts: BuildOptions): Promise<ReleasePromptAn
     },
   ];
 
-  let answers: ReleasePromptAnswers;
   try {
-    answers = await inquirer.prompt<ReleasePromptAnswers>(prompts);
-    opts.otp = answers.otp;
+    const answers = await inquirer.prompt<ReleasePromptAnswers>(prompts);
+    return {
+      confirm: answers.confirm,
+      otp: answers.otp,
+      npmTag: determineAnsweredTagToUse(answers),
+    };
   } catch (err: any) {
     console.log('\n', color.red(err), '\n');
     process.exit(0);
   }
-  return answers;
+}
+
+/**
+ * Helper function to determine which tag string to use.
+ *
+ * Due to a bug in Inquirer, the tag to publish Stencil under needs to be specified under two different fields when
+ * the release scripts diverge in questioning. This function ensures the logic for determining which of those two
+ * answers is to be used.
+ *
+ * @param answers user provided answers to pick from
+ * @returns the tag to use. defaults to 'UNKNOWN' if the tag cannot be determined.
+ */
+export function determineAnsweredTagToUse(answers: ReleasePromptAnswers): string {
+  return answers.tag ? answers.tag : answers.specifiedTag ? answers.specifiedTag : 'UNKNOWN';
```

Comment:
Same idea here:
```suggestion
  return answers.tag ?? answers.specifiedTag ?? 'UNKNOWN';
```

Reply:
We could, although I don't think that necessarily handles the case where either is set as an empty string.